src/lib/camel-case-keys.js

```diff
@@ -1,5 +1,26 @@
-const { mapKeys, camelCase } = require('lodash');
+const { reduce, isArray, isPlainObject, camelCase } = require('lodash');
```

Comment:
have we tried https://www.npmjs.com/package/deep-map-keys?

This may be as simple as:
```
const deepMapKeys = require('deep-map-keys');
const { camelCase } = require('lodash');
exports.camelCaseKeys = obj => deepMapKeys(obj, camelCase);
```

Reply:
done